stubs/protobuf/google/protobuf/api_pb2.pyi

```diff
@@ -214,7 +214,7 @@ class Mixin(google.protobuf.message.Message):
     The mixin construct implies that all methods in `AccessControl` are
     also declared with same name and request/response types in
     `Storage`. A documentation generator or annotation processor will
-    see the effective `Storage.GetAcl` method after inheriting
+    see the effective `Storage.GetAcl` method after inherting
```

Comment:
Lol, did they deliberately introduce a typo or did we accidentally update this in our vendored copy without contributing the typo fix upstream?

Reply:
They deliberately re-introduced it! https://github.com/protocolbuffers/protobuf/commit/b81c127e94d953aa9be5c9f00a1ad323552bae0e

Reply:
Amazing! If this didn't have a citation I'd be certain this was an April Fool's :D

Reply:
Got to have copyright on those typo fixes